src/components/change-requests/change-request-details/change-request-details/change-request-details.tsx

```diff
@@ -167,7 +87,7 @@ const ChangeRequestDetails: React.FC<ChangeRequestDetailsProps> = ({
         }
       />
       {buildDetails(changeRequest)}
-      <PageBlock title={'Implemented Changes'} headerRight={<></>} body={<>list of changes</>} />
+      <ImplementedChanges />
```

Comment:
This change will impact what @RChandler234 is working on, so unless he wants you to make the change then let's not make this change, especially since it a small change.

Reply:
Yeah, I would appreciate it if we didn't make this change if possible because otherwise I'm going to have to move around all the code I've been working on. If this causes you a bunch of issues let me know, and I can work around it. Otherwise though, the code looks fine/ shouldn't conflict with my PR

Reply:
Gotcha, I'll revert this change and leave out separating the implemented changes as its own component for this PR and we can just make a ticket to have it separated out in the future potentially.